website/scripts/javadocs.sh

```diff
@@ -19,17 +20,23 @@ APACHE_SRC_FILES=`find $HERON_ROOT_DIR -path "*/org/apache/storm/*" -name "*.jav
 GEN_FILES=`find $GEN_PROTO_DIR -name "*.java"`
 
 rm -rf $JAVADOC_OUTPUT_DIR
-mkdir -p $JAVADOC_OUTPUT_DIR
+mkdir -p $JAVADOC_OUTPUT_DIR $JAVADOC_STATIC_OUTPUT_DIR
 
 BIN_JARS=`find $HERON_ROOT_DIR/bazel-heron/_bin/. -name "*\.jar" | tr '\n' ':'`
 GEN_JARS=`find $HERON_ROOT_DIR/bazel-genfiles/external/. -name "*\.jar" | tr '\n' ':'`
 SCRIBE_JARS=`find $HERON_ROOT_DIR/bazel-bin/. -name "libthrift_scribe_java.jar" | tr '\n' ':'`
 PROTO_JARS=`find $HERON_ROOT_DIR/bazel-bin/heron/proto/. -name "*\.jar" | tr '\n' ':'`
-CLOSURE_CLASSES="$HERON_ROOT_DIR/bazel-bin/heron/storm/src/java/_javac/storm-compatibility-java/libstorm-compatibility-java_classes/."
+CLOJURE_CLASSES="$HERON_ROOT_DIR/bazel-bin/heron/storm/src/java/_javac/storm-compatibility-java/libstorm-compatibility-java_classes/."
 
-export CLASSPATH=$BIN_JARS:$GEN_JARS:$SCRIBE_JARS:$PROTO_JARS:$CLOSURE_CLASSES
+export CLASSPATH=$BIN_JARS:$GEN_JARS:$SCRIBE_JARS:$PROTO_JARS:$CLOJURE_CLASSES
 
-$JAVADOC $FLAGS -d $JAVADOC_OUTPUT_DIR $GEN_FILES $HERON_SRC_FILES $BACKTYPE_SRC_FILES $APACHE_SRC_FILES
+$JAVADOC -quiet \
+  -windowtitle "Heron Java API" \
+  -doctitle "The Heron Java API" \
+  -overview $OVERVIEW_HTML_FILE \
+  -d $JAVADOC_OUTPUT_DIR $GEN_FILES $HERON_SRC_FILES $BACKTYPE_SRC_FILES $APACHE_SRC_FILES
+
+cp -rf $JAVADOC_OUTPUT_DIR $JAVADOC_STATIC_OUTPUT_DIR/api
```

Comment:
This is related to #798. It's a proposed (and hopefully temporary) solution
that will enable us to use Javadocs locally and on the live site without
issue.
On Wed, May 25, 2016 at 7:53 PM Bill Graham <notifications@github.com>
wrote:

> In website/scripts/javadocs.sh
> <https://github.com/twitter/heron/pull/800#discussion_r64685454>:
>
> >
> > -$JAVADOC $FLAGS -d $JAVADOC_OUTPUT_DIR $GEN_FILES $HERON_SRC_FILES $BACKTYPE_SRC_FILES $APACHE_SRC_FILES
> > +$JAVADOC -quiet \
> > +  -windowtitle "Heron Java API" \
> > +  -doctitle "The Heron Java API" \
> > +  -overview $OVERVIEW_HTML_FILE \
> > +  -d $JAVADOC_OUTPUT_DIR $GEN_FILES $HERON_SRC_FILES $BACKTYPE_SRC_FILES $APACHE_SRC_FILES
> > +
> > +cp -rf $JAVADOC_OUTPUT_DIR $JAVADOC_STATIC_OUTPUT_DIR/api
>
> why do we need 2 copies of the javadocs in multiple folders?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/twitter/heron/pull/800/files/8fde8341a3f8f1df41b109d1558404d0e7b6e78d#r64685454>
>